tower-web/src/app/modules/main/component/navbar/navbar.component.ts

```diff
@@ -32,5 +32,12 @@ export class NavbarComponent implements OnInit {
     )
   }
 
+  toggleSidebar() {
+    console.log(this);
+    this.isSidebarToggled = !this.isSidebarToggled;
+    // TODO: Make this work properly instead of hacking with jQuery
```

Comment:
Couldn't figure out how to do this nicely on the plane - just toggling a class on another element that's in a different scope? This works fine but is hacky..

Reply:
Looking at a couple of stackoverflow posts, it seems crazy complicated to do this with angular! Maybe my little one-liner jQuery hack isn't all that bad..?! 😆

Reply:
Hang on, I think this jQuery command stops the front end from building... 🤦‍♂ Have commented out in the last commit, so sidebar doesn't work now.

Reply:
Ouch! 😆

Reply:
Yes, I need to fix this 😅 Will let you know when it’s ready for testing / merging..

Reply:
Fixed! 🎉